ConcernsCaseWork/ConcernsCaseWork.Tests/Pages/HomePageModelTests.cs

```diff
@@ -46,8 +46,8 @@ public async Task WhenInstanceOfIndexPageOnGetAsync_ReturnCases()
 			var defaultUsers = new[] { "user1", "user2" };
 			var roleClaimWrapper = new RoleClaimWrapper { Roles = roles, Users = defaultUsers };
 
-			mockRbacManager.Setup(r => r.GetUserRoleClaimWrapper(It.IsAny<string>()))
-				.ReturnsAsync(roleClaimWrapper);
+			//mockRbacManager.Setup(r => r.GetUserRoleClaimWrapper(It.IsAny<string>()))
```

Comment:
Would be good to know why this is commented out (normally I'd reject commented out code, but I guess this is because we're in a state of flux)

Reply:
Not used anymore, roles are going to come from the bearer token, using the ClaimsPrincipalHelper. Will remove that comment.